kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,8 +1,7 @@
 """Project settings."""
-from {{cookiecutter.python_package}}.hooks import ProjectHooks
 
-# Instantiate and list your project hooks here
-HOOKS = (ProjectHooks(),)
+# Instantiate and list your custom project hooks here
+# HOOKS = (ProjectHooks(),)
```

Comment:
I think we should keep this line in, just commented out. Otherwise the line with `ProjectHooks()` is kind of mysterious because it's not clear where it comes from.
```suggestion
# Instantiate and list your custom project hooks here
# from {{cookiecutter.python_package}}.hooks import ProjectHooks
# HOOKS = (ProjectHooks(),)
```

Reply:
We could probably make these explanations (e.g. about where to define your hooks) a bit clearer but I would leave that for another PR. As per a comment I made on another PR, we should go through settings.py after all this is done and make sure it's all clear and consistent 🙂